src/components/entries/Tooltip.js

```diff
@@ -107,7 +107,9 @@ function Tooltip(props) {
       onMouseEnter={ showTooltip }
       onMouseLeave={ ()=> clearTimeout(timeout) }
       onFocus={ showTooltip }
-      onKeyDown={ hideTooltipViaEscape }>
+      onKeyDown={ hideTooltipViaEscape }
+      onMouseDown={ (e)=> {e.preventDefault();} }
```

Comment:
prevents tooltip from staying open after opening to click group

Reply:
Doesn't seem to work for me.

![brave_DS1UsEqP8r](https://github.com/bpmn-io/properties-panel/assets/7633572/4cb1c1e9-e6ac-41a1-8c7a-5e3ee5c7a019)

Reply:
Resolved ✅